Doc/library/datetime.rst

```diff
@@ -1761,17 +1761,17 @@ is aware, :exc:`TypeError` is raised if an order comparison is attempted. For eq
 comparisons, naive instances are never equal to aware instances.
 
 If both comparands are aware, and have
-the same :attr:`~time.tzinfo` attribute, the common :attr:`~time.tzinfo` attribute is
+the same :attr:`~.time.tzinfo` attribute, the common :attr:`!tzinfo` attribute is
 ignored and the base times are compared. If both comparands are aware and
-have different :attr:`~time.tzinfo` attributes, the comparands are first adjusted by
+have different :attr:`!tzinfo` attributes, the comparands are first adjusted by
 subtracting their UTC offsets (obtained from ``self.utcoffset()``). In order
 to stop mixed-type comparisons from falling back to the default comparison by
 object address, when a :class:`.time` object is compared to an object of a
 different type, :exc:`TypeError` is raised unless the comparison is ``==`` or
 ``!=``. The latter cases return :const:`False` or :const:`True`, respectively.
 
 .. versionchanged:: 3.3
-  Equality comparisons between aware and naive :class:`~datetime.time` instances
+  Equality comparisons between aware and naive :class:`~.datetime.time` instances
```

Comment:
The same as above. `~.datetime.time` looks suspicious to me. I think that `.time` or `~.time` should work.